example/server/integrations/deposit.py

```diff
@@ -51,27 +81,73 @@ def content_for_template(
         **kwargs,
     ) -> Optional[Dict]:
         na, kyc_content = SEP24KYC.check_kyc(transaction)
-        if kyc_content:
+        if kyc_content:  # the user needs to confirm their email (or skip confirmation)
             return kyc_content
         elif template == Template.DEPOSIT:
-            if not form:
+            if not form:  # we're done
                 return None
-            return {
-                "title": _("Polaris Transaction Information"),
-                "guidance": _("Please enter the amount you would like to transfer."),
-                "icon_label": _("Stellar Development Foundation"),
-            }
+            elif isinstance(form, SelectAssetForm):
+                return {
+                    "title": _("Asset Selection Form"),
+                    "guidance": _(
+                        "Please select the asset you would like to "
+                        "provide in order to fund your deposit."
+                    ),
+                    "icon_label": _("Stellar Development Foundation"),
+                }
+            elif isinstance(form, TransactionForm) or isinstance(
+                form, OffChainAssetTransactionForm
+            ):
+                return {
+                    "title": _("Transaction Amount Form"),
+                    "guidance": _("Please enter the amount you would like to provide."),
+                    "icon_label": _("Stellar Development Foundation"),
+                }
+            elif isinstance(form, ConfirmationForm):
+                return {
+                    "title": _("Transaction Confirmation Page"),
+                    "guidance": _("Review and confirm the transaction details"),
+                    "icon_label": _("Stellar Development Foundation"),
+                    "template_name": "transaction_confirmation.html",
+                    "amount_in": transaction.amount_in,
+                    "amount_fee": transaction.amount_fee,
+                    "conversion_amount": round(
+                        transaction.amount_in - transaction.amount_fee,
+                        transaction.asset.significant_decimals,
+                    ),
+                    "amount_out": transaction.amount_out,
+                    "amount_in_symbol": "USD",
+                    "amount_fee_symbol": "USD",
+                    "amount_out_symbol": transaction.asset.symbol,
+                    "amount_in_significant_decimals": 2,
```

Comment:
May be beneficial to sync up with the SEP-1's `display_decimals` or make them configurable. 

But I think it may be fine since this is for anchor's reference.

Reply:
I need to use `OffChainAsset.significant_decimals`